german/finger-ins-loch/hoelzernes-Paddel/sicherer_Entfernung/gruessen_freundlich/gruessen_freundlich.md

```diff
@@ -5,3 +5,7 @@ ihr Gesicht geschrieben. Gerade als Sie ...
 "Du darfst nicht hier sein!", zischt Ihnen die junge Frau zu. Feuchte, dunkle Haarsträhnen hängen
 ihr ins Gesicht, darunter blitzen merkwürdig schwarze Augen im Feuerschein. Etwas scheint außerdem
 mit dem Mann nicht zu stimmen. Nach einem Augenblick wird Ihnen klar, dass er blind ist.
+
+Sie entschließen sich 
+
+[dazu vor den beiden abzuhauen](https://github.com/ll9/create-your-own-adventure/blob/some-changes/german/finger-ins-loch/hoelzernes-Paddel/sicherer_Entfernung/gruessen_freundlich/abhauen/abhauen.md)
```

Comment:
This link seems to be an absolute link to GitHub rather than a relative link.  If you are linking to another page in the create-your-own-adventure repository, please use relative links to do so.  See the README of this repository for more information.